spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/command/CommandRunner.java

```diff
@@ -11,18 +25,18 @@
  *     <li>convert exception into a {@link Result}, increase error count</li>
  * </ol>
  */
-public class CommandRunner {
-    private final CommandHandlerRegistry commandHandlerRegistry;
-    private final Monitor monitor;
+public class CommandRunner<C extends Command> {
+    protected final CommandHandlerRegistry<C> commandHandlerRegistry;
```

Comment:
Why does this need to be `protected`?

Reply:
It doesn't. CommandRunner was an abstract class in between, missed it when I refactored it back. Changed it now.